src/model/Family.model.js

```diff
@@ -0,0 +1,13 @@
+import Task from '@/model/Task.model'
+
+export default class Family {
+  constructor (name, cyclePoint, state, depth, childTasks) {
+    this.name = name
+    this.cyclePoint = cyclePoint
```

Comment:
Why do we need a cycle point for a family? I guess this is a family of task proxies instead of a family task definitions?

Would it be clearer to call it `FamilyProxy` (and `TaskProxy`)?

Reply:
>Why do we need a cycle point for a family? I guess this is a family of task proxies instead of a family task definitions?

Question for https://github.com/cylc/cylc-flow/pull/3122 I think.

>Would it be clearer to call it FamilyProxy (and TaskProxy)?

I think it could be... but a `FamilyProxy` has `childTasks` and `childFamilies`... which I am not sure if they are proxies too or not... also not clear whether we will have `Family` anywhere else, or if the `FamilyProxy` is supposed to represent the `Family` for the UI... in that case I think `Family` would be easier to understand in the UI... but maybe we can leave this one to be discussed later?

Reply:
Yes. We should clarify this after merging @dwsutherland GraphQL changes in the other projects..